modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMAnnouncements.macro

```diff
@@ -0,0 +1,18 @@
+definition {
+
+	@summary = "Add Announcement and Publish in page the PRM"
+	macro addAndPublishAnnouncement {
+		LexiconEntry.gotoAddMenuItem(menuItem = "Announcement");
+
+		PortletEntry.inputTitle(title = "Announcement Title");
+
+		CKEditor.addContentInWebContent(
+			content = "Announcement Body",
+			fieldLabel = "Body");
```

Comment:
Hi @marinaasant0s, if you add a variable on Announcement Title and Body, the macro could be more generic and useful. What do you think? It could be "${announcementTitle}" and "${announcementBody}", so you can define the title and body on the test, where you call the macro.